test/runtime/Policies.go

```diff
@@ -202,7 +202,7 @@ var _ = Describe("RuntimePolicyEnforcement", func() {
 			ExpectEndpointSummary(vm, helpers.Enabled, 0)
 			ExpectEndpointSummary(vm, helpers.Disabled, 1)
 
-			vm.ContainerCreate("new", "cilium/demo-httpd", helpers.CiliumDockerNetwork, "-l id.new")
+			vm.ContainerCreate("new", "docker.io/cilium/demo-httpd:latest", helpers.CiliumDockerNetwork, "-l id.new")
```

Comment:
now that you change this one, could you use `helpers.HttpdImage`

Reply:
Done